edit/edit.go

```diff
@@ -392,19 +393,22 @@ func move(ed *Editor, src, dst Address) (addr, error) {
 // It is an error if such a subexpression has more than MaxRunes.
 // \n is a literal newline.
 // If g is true then all matches in the address range are substituted.
+// The n-th address in the range is substituted.
+// If n is not equal to '1' g must be false.
 // Dot is set to the modified address.
-func (ed *Editor) Substitute(a Address, re *re1.Regexp, repl []rune, g bool) error {
-	return ed.do(func() (addr, error) { return sub(ed, a, re, repl, g) })
+func (ed *Editor) Substitute(a Address, re *re1.Regexp, repl []rune, g bool, n int) error {
```

Comment:
We talked about having SubstituteFrom which includes `n` and Substitute which has a default of `n==1`.